### PR TITLE
Jenish

### PR DESCRIPTION
**Pull request by Jenishleap**

### CODE REVIEW

File: bigsmile.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Big smile jenish 
 shaky ji from Patna
-
+I made a new branch jenish
```
Comment (subash-poudel) [inline bigsmile.txt]: Why???

Comment (Jenishleap) [reply bigsmile.txt]: fixed merged conflict

Comment (Jenishleap) [reply bigsmile.txt]: :ok:

**Comment by subash-poudel** [comment]
👍

**Pull request merged**